appcenter-push/android/build.gradle

```diff
@@ -21,11 +21,7 @@ dependencies {
     implementation ('com.facebook.react:react-native:+') {
         exclude group: 'com.android.support'
     }
-    api ('com.microsoft.appcenter:appcenter-push:2.0.0') {
-        exclude group: 'com.google.firebase'
-    }
-    api 'com.google.firebase:firebase-core:16.0.4'
-    api 'com.google.firebase:firebase-messaging:17.3.4'
+    api 'com.microsoft.appcenter:appcenter-push:2.1.0-7+df4dfd7f7'
```

Comment:
This works and will be overridden during the release process to `2.1.0` so it's fine to commit prerelease.

Reply:
Thanks for checking the release automation script, I will also keep an eye on it in the release PR.